### PR TITLE
fix: some missing Fars

### PR DESCRIPTION
**Pull request by erights**

### CODE REVIEW

File: packages/SwingSet/test/gc/bootstrap.js
Comment (erights) [inline packages/SwingSet/test/gc/bootstrap.js]: Reviewers: What should this be called?

File: packages/governance/src/ballotBuilder.js
Comment (erights) [inline packages/governance/src/ballotBuilder.js]: Reviewers: What should this be called?

File: packages/SwingSet/test/gc/bootstrap.js
Comment (warner) [reply packages/SwingSet/test/gc/bootstrap.js]: I usually call it `root`.

**Review by warner — APPROVED** [review]
LGTM, regardless of what names you end up using

File: packages/governance/src/ballotBuilder.js
Comment (Chris-Hibbert) [reply packages/governance/src/ballotBuilder.js]: ballot details

**Review by Chris-Hibbert — APPROVED** [review]

**Comment by erights** [comment]
Hi @warner although you and @Chris-Hibbert already approved when the PR was about 3 lines, and although the changes since then are largely mechanical, since the changes ballooned, I prefer to wait until you look again before merging. Thanks.

**Comment by erights** [comment]
Hi @Chris-Hibbert , due to https://github.com/Agoric/agoric-sdk/pull/3498/commits/f5dd2b8cc695cca4a93f73b6761df60c145fa1a0 , PTAL.

**Review by warner — APPROVED** [review]
Those all look great

**Comment by erights** [comment]
@Chris-Hibbert I just need you to look at names starting with commit https://github.com/Agoric/agoric-sdk/pull/3498/commits/857c5ac30e028d6acd83965a2bd6abb2cf328b59 "More far straglers" (sic). 

Please tell me what names I should use instead. Thanks.

File: packages/treasury/src/liquidateMinimum.js
Comment (Chris-Hibbert) [inline packages/treasury/src/liquidateMinimum.js]:
```suggestion
  const creatorFacet = Far('debtorInvitationCreator', {
```

File: packages/treasury/src/stablecoinMachine.js
Comment (Chris-Hibbert) [inline packages/treasury/src/stablecoinMachine.js]:
```suggestion
  const stablecoinMachine = Far('stablecoin machine', {
```

File: packages/treasury/src/vaultManager.js
Comment (Chris-Hibbert) [inline packages/treasury/src/vaultManager.js]: Shouldn't need a `Far` declaration. The only uses are with spread operator to add these operations to other `Far` objects.  (okay, and one direct use, but it never escapes this file.)

File: packages/treasury/src/vaultManager.js
Comment (Chris-Hibbert) [inline packages/treasury/src/vaultManager.js]: @katelynsills   are invitationMakers used with the wallet?  Does it need their keys to be Keyword-style?

File: packages/zoe/tools/priceAuthorityRegistry.js
Comment (Chris-Hibbert) [inline packages/zoe/tools/priceAuthorityRegistry.js]:
```suggestion
  const priceAuthority = Far('fake price authority', {
```

**Review by Chris-Hibbert — APPROVED** [review]
Added some naming suggestions. 

One internal structure doesn't seem like it needs `Far`. 

I referred one declaration to @katelynsills as it looks like it might be a wallet thing.

File: packages/treasury/src/vaultManager.js
Comment (erights) [reply packages/treasury/src/vaultManager.js]: correct. fixed

File: packages/treasury/src/vaultManager.js
Comment (erights) [reply packages/treasury/src/vaultManager.js]: Given https://github.com/Agoric/agoric-sdk/issues/3503 and a plausible name, I'm resolving this.

Comment (katelynsills) [reply packages/treasury/src/vaultManager.js]: Yup, invitationMakers are used within dapp-svelte wallet and they need to be Keyword-style so that the wallet is protected against a contract developer.

**Pull request merged**
